Gemfile

```diff
@@ -5,7 +5,7 @@ gemspec
 
 group :test do
   gem 'actionpack'
-  gem 'logstash-event', '1.1.5'
+  gem 'logstash-event', git: 'https://github.com/elasticsearch/logstash.git', tag: '1.5'
```

Comment:
What's the rationale for pinning to that particular tag? If this is completely necessary we should document this I guess.